src/services/uploadAttachment.ts

```diff
@@ -39,5 +39,6 @@ export default async function uploadAttachment(
     fileName,
     id: formAttachmentS3Credentials.attachmentDataId,
     isPrivate,
+    uploadedAt: Date.now().toString(),
```

Comment:
This value is actually return from the request for credentials, the reason we take that and not the value from the client is because we cannot rely on the client having the correct date/time set on their machine.